lib/services/sku-pack-service.js

```diff
@@ -38,7 +44,10 @@ function skuPackServiceFactory(
     assert,
     workflowApiService,
     Constants,
-    Env
+    Env,
```

Comment:
Hi Flix, it is written throughout the code Env and Constant with upper-case E and C, so i am  not going to change just to avoid confusion.